### DISCUSSION TITLE
Avoiding noise from bots in github release changelog

### DISCUSSION THREAD

**Discussion by ssbarnea** [tox-dev/tox#3169] [Ideas]
I checked https://github.com/tox-dev/tox/releases page and I find quite hard to read what really matters due to the noise we got from, mostly, pre-commit auto-updates.

I am not sure how this changelog is generated, but it would be very useful to remove the bots from it. I know how to do this with release-drafter but tox is using another system to update this.

**Comment by gaborbernat** [comment]
This was already done via https://github.com/tox-dev/tox/blob/main/.github/release.yml#L1 added 2 weeks ago.

**Reply by gaborbernat** [reply]
https://tox.wiki/en/4.11.4/changelog.html is the better way to view changelogs 😊 but we also offer GitHub releases as an extra.
